services/app-graphql/src/schema.graphql

```diff
@@ -318,7 +340,7 @@ input CreateQuestionInput {
     "The ID of the package for which to create a question"
     pkgID: ID!
     "A list of documents to attach to the question"
-    documents: [DocumentInput!]!
+    documents: [DocumentInput!]! # should be Generic Document with sha256
```

Comment:
Yes I agree.  was thinking  everywhere in graphql could use my new `GenericDocument` eventually. @macrael Does that sound fine?

  I did start to just make change monday, adding sha2556 to Q&A stuff, but quickly ran into types issues because there were more places using that type in small utility functions and other places than I anticipated. I backtracked then and just left the comment.